deploy/apparmor/libvirt-qemu

```diff
@@ -0,0 +1,224 @@
+# Last Modified: Wed Sep 3 21:52:03 2014
```

Comment:
As commented earlier, consider having a versioned folder for these published configurations.

Reply:
yes, let's get a versioning story post 330 release.